corehq/apps/app_manager/templates/app_manager/v1/form_designer.html

```diff
@@ -17,6 +17,9 @@
 {% block js %}{{ block.super }}
     <script src="{% static 'moment/moment.js' %}"></script>
     <script src="{% static 'requirejs/require.js' %}"></script>
+    {% if not request|toggle_enabled:'VELLUM_BETA' %}
```

Comment:
at one point we used to do it for all our js blocks (from our main style base.html) however that cause so many compression errors like these we got rid of it haha